src/containers/container-recently-todo/goal-list-component.tsx

```
@@ -0,0 +1,29 @@
+import { Todo } from "@/types/container-recently-todo";
+
+import Goal from "./goal";
```

ESLint

[error] Missing file extension for "./goal"

(import/extensions)